docs/content/policy-reference.md

```diff
@@ -899,6 +899,86 @@ The table below shows examples of calling `http.send`:
 | Environment variables containing TLS material | ``http.send({"method": "get", "url": "https://127.0.0.1:65360", "tls_ca_cert_env_variable": "CLIENT_CA_ENV", "tls_client_cert_env_variable": "CLIENT_CERT_ENV", "tls_client_key_env_variable": "CLIENT_KEY_ENV"})`` |
 | Unix Socket URL Format| ``http.send({"method": "get", "url": "unix://localhost/?socket=%F2path%F2file.socket"})`` |
 
+{{< builtin-table cat=providers.aws title=AWS >}}
+
+The AWS Request Signing builtin in OPA implements the header-based auth,
+single-chunk method described in the [AWS SigV4 docs](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-header-based-auth.html).
+It will always sign the payload when present, and will sign most user-provided
+headers for the request, to ensure their integrity.
+
+{{< info >}}
+Note that the `authorization`, `user-agent`, and `x-amzn-trace-id` headers,
+are commonly modified by proxy systems, and as such are ignored by OPA
+for signing.
+{{< /info >}}
+
+The `request` object parameter may contain any and all of the same fields as for `http.send`.
+The following fields will have effects on the output `Authorization` header signature:
+
+| Field | Required | Type | Description |
+| --- | --- | --- | --- |
+| `url` | yes | `string` | HTTP URL to specify in the request. Used in the signature. |
+| `method` | yes | `string` | HTTP method to specify in request. Used in the signature. |
+| `body` | no | `any` | HHTTP message body. The JSON serialized version of this value will be used for the payload portion of the signature if present. |
+| `raw_body` | no | `string` | HTTP message body. This will be used for the payload portion of the signature if present. |
+| `headers` | no | `object` | HTTP headers to include in the request. These will be added to the list of headers to sign. |
+
+The `aws_config` object parameter may contain the following fields:
+
+| Field | Required | Type | Description |
+| --- | --- | --- | --- |
+| `aws_access_key` | yes | `string` | AWS access key. |
+| `aws_secret_access_key` | yes | `string` | AWS secret access key. Used in generating the signing key for the request. |
+| `aws_service` | yes | `string` | AWS service the request will be valid for. (e.g. `"s3"`) |
+| `aws_region` | yes | `string` | AWS region for the request. (e.g. `"us-east-1"`) |
+| `aws_session_token` | no | `string` | HTTP headers to include in the request. These will be added to the list of headers to sign. |
```

Comment:
This description seems misplaced.

Reply:
You are correct! That was a copy/paste error on my part. :sweat_smile: